client/my-sites/plans/wx-medium/index.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+	getPlans,
+	PLAN_WOOEXPRESS_MEDIUM,
+	PLAN_WOOEXPRESS_MEDIUM_MONTHLY,
+} from '@automattic/calypso-products';
+import { Button, Card } from '@automattic/components';
+import { formatCurrency } from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import getInTouch from 'calypso/assets/images/plans/wpcom/get-in-touch.png';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import { SitePlanData } from 'calypso/my-sites/checkout/composite-checkout/hooks/product-variants';
+import { getPlanRawPrice, getPlan } from 'calypso/state/plans/selectors';
+import TrialFeatureCard from '../components/ecommerce-plan-features/trial-feature-card';
+import { getWooExpressMediumFeatureSets } from '../ecommerce-trial/wx-medium-features';
+
+import './style.scss';
+
+interface WXMediumPlansPageProps {
+	currentPlan: SitePlanData;
+}
+
+const WXMediumPlansPage = ( { currentPlan }: WXMediumPlansPageProps ) => {
```

Comment:
Nit: I'd strongly prefer we call this `WooExpressMediumPlansPage` - the `WX` abbreviation is going to be hard to find for people maintaining this code.